[PLATFORM-509]: [Auth0Ex] More generic Absinthe plug (#94)
primait/auth0_ex: CHANGELOG.md

## [Next]

### Changed

> - Absinthe plugs do not use a custom context structor but instead use a map

Reviewer: ```suggestion
- Absinthe plugs use a map instead of custom struct as context
```
